docker/mercury_inventory/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM centos/python-36-centos7:1
+# for lack of a better e-mail...just using a dev on the team ;)
+MAINTAINER DCX Baremetal Devs <ben.meyer@rackspace.com>
+
+ENV APP_NAME mercury-inventory
+ENV SOURCE_ROOT ${HOME}/${APP_NAME}
+ENV LOG_DIR /var/log/${APP_NAME}
+ENV PID_DIR /var/run/${APP_NAME}
+ENV MERCURY_CONFIG_DIR ${HOME}/.mercury
+SHELL ["/bin/bash", "-c"]
+
+VOLUME ${HOME}/${APP_NAME}
+
+# switch to user in order to update the base image
+USER root
+# IPv6 doesn't resolve over VPN at least so force IPv4
+RUN echo "ip_resolve=4" >> /etc/yum.conf
+RUN yum -y update && yum -y install @development-tools fedora-packager rpmdevtools
+
+# Create log & pid directories
+RUN mkdir -p ${LOG_DIR} && chmod -R +w ${LOG_DIR} && chown -R default:root ${LOG_DIR}
+RUN mkdir -p ${PID_DIR} && chmod -R +w ${PID_DIR} && chown -R default:root ${PID_DIR}
+
+# switch back to the `default` user used for the Application by the base image
+USER default
+RUN pip install --upgrade pip
+
+
+ADD . ${HOME}/${APP_NAME}
+
+WORKDIR ${SOURCE_ROOT}
+RUN cd src/mercury-common && pip install -r test-requirements.txt && pip install -e .
+RUN cd src/${APP_NAME} && pip install -e .
+
+RUN scripts/make_docker_configs.sh --rebuild inventory --database "mongo:27017" --inventory-router "tcp://mercury_inventory:9000"
+EXPOSE 9000:9000
+CMD python src/mercury-inventory/mercury/inventory/server.py
```

Comment:
Since PR #62 is merged, a console script is now available: `mercury-inventory`, which supports the following command line options:

```
usage: mercury-inventory [-h] [-c CONFIGURATION] [--log-level LOG_LEVEL]
                         [--log-format LOG_FORMAT]
                         [--inventory-bind-address INVENTORY_BIND_ADDRESS]
                         [--asyncio-debug ASYNCIO_DEBUG]
                         [--inventory-db-name INVENTORY_DB_NAME]
                         [--inventory-db-collection INVENTORY_DB_COLLECTION]
                         [--inventory-db-servers INVENTORY_DB_SERVERS]
                         [--inventory-db-replica-name INVENTORY_DB_REPLICA_NAME]

optional arguments:
  -h, --help            show this help message and exit
  -c CONFIGURATION, --configuration CONFIGURATION
                        Path to the configuration file
  --log-level LOG_LEVEL
                        Log level, INFO | DEBUG | ERROR | WARNING
  --log-format LOG_FORMAT
                        Desired logging format
  --inventory-bind-address INVENTORY_BIND_ADDRESS
                        The address to bind to
  --asyncio-debug ASYNCIO_DEBUG
                        Enable asyncio debugging
  --inventory-db-name INVENTORY_DB_NAME
                        The database for our collections
  --inventory-db-collection INVENTORY_DB_COLLECTION
                        The collection for our documents
  --inventory-db-servers INVENTORY_DB_SERVERS
                        Server or coma separated list of servers to connect to
  --inventory-db-replica-name INVENTORY_DB_REPLICA_NAME
                        An optional replica
```

Reply:
in-progress